src/compile_yara_signatures.py

```diff
@@ -48,7 +48,7 @@ def _create_compiled_signature_file(directory, tmp_file):
     target_path = os.path.join(SIGNATURE_DIR, '{}.yc'.format(_get_plugin_name(directory)))
     try:
         command = 'yarac -d test_flag=false {} {}'.format(tmp_file.name, target_path)
-        execute_shell_command(command, check=True)
+        subprocess.run(command, stdout=PIPE, stderr=STDOUT, shell=True, check=True)
```

Comment:
It could also make sense to replace some other calls where we a primarily interested in the output with `subprocess.check_output`, because it is usually shorter.

Reply:
`subprocess.{check_call,check_output}` are listed as `Older high-level API` in the docs.
Should we still use them?

Reply:
I did not notice that :sweat_smile: 
If that is the case, then it makes sense to only use `run` (though I doubt that the functions will be deprecated anytime soon)